src/NzbDrone.Core.Test/HealthCheck/Checks/ImportListStatusCheckFixture.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using Moq;
+using NUnit.Framework;
+using NzbDrone.Core.HealthCheck.Checks;
+using NzbDrone.Core.ImportLists;
+using NzbDrone.Core.Test.Framework;
+
+namespace NzbDrone.Core.Test.HealthCheck.Checks
+{
+    [TestFixture]
+    public class ImportListStatusCheckFixture : CoreTest<ImportListStatusCheck>
+    {
+        private List<IImportList> _importLists = new List<IImportList>();
+        private List<ImportListStatus> _blockedImportLists = new List<ImportListStatus>();
+
+        [SetUp]
+        public void SetUp()
+        {
+            Mocker.GetMock<IImportListFactory>()
+                  .Setup(v => v.GetAvailableProviders())
+                  .Returns(_importLists);
+
+            Mocker.GetMock<IImportListStatusService>()
+                   .Setup(v => v.GetBlockedProviders())
+                   .Returns(_blockedImportLists);
+        }
+
+        private Mock<IImportList> GivenImportList(int i, double backoffHours, double failureHours)
+        {
+            var id = i;
```

Comment:
Rename `i` to `id`

Reply:
Will fix the IndexerStatusCheckFixture as well